lib/eco.js

```diff
@@ -8,7 +8,7 @@ class SimplyEco {
 	 *@param dbUrl
 	 *@param options
 	 */
-	constructor(dbUrl, options = []) {
+	constructor(dbUrl, options = {}) {
```

Comment:
If needed is a object then it would be the best to have the default options as a object as well.

Reply:
Yeah i understand that, it was a typing error